tophat/django-training
# docs: Add website

## QA

1. `cd website`
1. `yarn`
1. Verify that an empty website shows up

Will add the website build and deploy to CI in a follow-up PR
